app/src/main/res/layout/message_activity.xml

```diff
@@ -3,15 +3,15 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:padding="16dp">
+    android:padding="@dimen/padding">
```

Comment:
padding what? It's too generic

Reply:
Ohk got it... I thought since there is just one screen currently and the padding is same so that's y but got it I'll do the changes